packages/react-code-editor/package.json

```
@@ -30,10 +30,10 @@
     "clean": "rimraf dist"
   },
   "dependencies": {
-    "@patternfly/react-core": "^4.251.1",
-    "@patternfly/react-icons": "^4.92.6",
-    "@patternfly/react-styles": "^4.91.6",
-    "react-dropzone": "^14.2.3",
+    "@patternfly-react/react-core": "^1.0.0",
+    "@patternfly-react/react-icons": "^1.0.0",
+    "@patternfly-react/react-styles": "^1.0.0",
```

Should these be `@patternfly-five` as well? It looks like the scope `@patternfly-react` doesn't exist on npm... When I install I get an error for each of those instances:

```
error Couldn't find package "@patternfly-react/react-core@^1.0.0" required by "@patternfly-five/react-code-editor@1.0.0" on the "npm" registry.
```

```suggestion
    "@patternfly-five/react-core": "^1.0.0",
    "@patternfly-five/react-icons": "^1.0.0",
    "@patternfly-five/react-styles": "^1.0.0",
```